_translator-files/po/sv-SE/FAQ.po

```diff
@@ -70,165 +71,167 @@ msgid ""
 #, markdown-text
 msgid "Whether or not you will be able to play in time with other musicians depends mainly on how much latency (delay) you have in your sound signal and whether you are all following [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together). An overall delay much bigger than 50 ms usually makes it too hard to play in time unless you train yourself to do so. Some musicians report they can play in time with delays of up to 70 ms by playing ahead of the beat. But much beyond that and it becomes too difficult unless timing isn’t an important factor in the music or you just want to listen."
-msgstr ""
+msgstr "Om du kan spela i takt med andra musiker beror främst på hur mycket latens (fördröjning) du har i ditt ljudsignal och om ni alla följer [Den gyllene regeln](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together). En total fördröjning som är mycket större än 50 ms gör det vanligtvis för svårt att spela i takt, om du inte tränar dig själv att göra det. Vissa musiker rapporterar att de kan spela i takt med fördröjningar på upp till 70 ms genom att spela före takten. Men mycket mer än så blir det för svårt om timing inte är en viktig faktor i musiken eller om du bara vill lyssna."
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:40
 #, markdown-text
 msgid "So it makes sense to minimise any sources of delay or other problems with the signal. Even fast wi-fi is usually too variable to use for long periods with Jamulus, and Bluetooth headphones and mics usually introduce latencies of about 50 ms or more."
-msgstr ""
+msgstr "Så det är rimligt att minimera alla källor till fördröjning eller andra problem med signalen. Även snabb Wi-Fi är vanligtvis för variabel för att användas under längre perioder med Jamulus, och Bluetooth-hörlurar och mikrofoner introducerar vanligtvis latenser på cirka 50 ms eller mer."
 
 #. type: Title ###
 #: ../wiki/en/FAQ.md:41
 #, markdown-text, no-wrap
 msgid "Why should I not listen to my own signal?"
-msgstr ""
+msgstr "Varför bör jag inte lyssna på mitt eget ljud?"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:44
 #, markdown-text
 msgid "For the same reason as you need to minimise delay in your signal in order to play in time, you need to make sure you are playing to your own sound that other musicians are hearing. More information on this, and a way of testing your setup to make sure you are obeying this “Golden Rule” [can be found here](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together)."
-msgstr ""
+msgstr "Av samma anledning som du behöver minimera fördröjning i din signal för att spela i takt, måste du se till att du spelar till ditt eget ljud som de andra musikerna hör. Mer information om detta och ett sätt att testa din setup för att säkerställa att du följer denna \"gyllene regel\" [kan du hitta här](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together)."
```

Comment:
Translate text after # in link.

Reply:
`ni-låter-alla-okej-men-det-är-svårt-att-hålla-ihop`